solr/server/etc/security.policy

```diff
@@ -191,6 +191,12 @@ grant {
   permission java.io.FilePermission "${solr.jetty.truststore}", "read,write,delete,readlink";
   permission java.io.FilePermission "${solr.jetty.truststore}${/}-", "read,write,delete,readlink";
 
+  permission java.io.FilePermission "${javax.net.ssl.keyStore}", "read,write,delete,readlink";
```

Comment:
is write and delete needed? I'm assuming no and only read is needed.

I also don't know if the `/` directory version is needed. Isn't `javax.net.ssl.*Store` going to point directly to the file in question?

Reply:
That's fair. I was just copying from above. Will fix.